recipe/meta.yaml

```diff
@@ -33,7 +33,7 @@ build:
 outputs:
   - name: cuda-nvcc-tools
     files:                                  # [linux]
-      - bin                                 # [linux]
+      - bin/*                               # [linux]
```

Comment:
These should be equivalent. Was there some issue that we encountered that motivated this change

Reply:
They are not equivalent, this will not copy directories (so everything except `crt`). Note the behavior here https://github.com/conda-forge/cuda-cudart-feedstock/pull/4

Reply:
Gotcha, thanks Alex! 🙏

Wonder if we should use [recursive globbing]( https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#build-section )

```suggestion
      - bin/**                              # [linux]
```

Reply:
Recursion is what we don't want here. The current change installs everything except `bin/crt`, which is moved to [another output](https://github.com/conda-forge/cuda-nvcc-impl-feedstock/pull/8/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR184-R185).

Reply:
Ok, thanks for clarifying